ushahidi/platform-client-mzima
# 1023 import error messages

Fixes [USH-1023](https://linear.app/ushahidi/issue/USH-1023/imports-error-on-mismatched-fields-fe).

This PR adjusts the error-message shown to the user when something goes wrong with an import, after mapping fields in a survey with columns in a csv.

To test:
1. Have a test-csv handy with known mismatch for a field, for example text for a number-field, I used this:

```
Title,Description,Number
Hello,Listing 1,1  
Hi,Listing 2,2  
Hello2,Listing 3,3  
Error ,Listing 4,"4"

```
2. Go to Settings and select "Import"
3. Select the csv-file and chose a survey with a number-field
4. Map the fields in the form 
5. Click import
-> The error-message appears. 


Note: This was the best I managed to do with the information I got from the API, sub-issue to fix the error-messages from the API: https://linear.app/ushahidi/issue/USH-1076/adjust-error-messages-from-the-api-for-csv-imports.
